assignments/prototypes.js

```diff
@@ -7,41 +7,46 @@
   
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
-  
-/*
-  === GameObject ===
-  * createdAt
-  * dimensions (These represent the character's size in the video game)
-  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
-*/
+ 
+  function GameObject () {
```

Comment:
your game object should be taking in some attributes as a param to bind values such as the dimensions that you added below, this could be written as 
`function GameObject(attributes) {
this.dimension = attributes.dimension`
you can then define the values of the dimension when you create your character as a new instance of the game object.